COMPONENTS/avatar/index.html

```diff
@@ -0,0 +1,17 @@
+<link
+  rel="stylesheet"
+  href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+/>
+<link rel="stylesheet" href="./index.css" />
+
+<div class="container p2">
+  <div class="avatar">
+    <div class="avatar_div t3 br-full" color="rgba(255, 59, 48)">
+      <!-- <i class="far fa-envelope-open"></i> -->
+      <!-- HF -->
+      <img alt="" src="https://picsum.photos/200" />
```

Comment:
give alt attribute a value specific to the image

Reply:
the reason I have left it empty is that by default it takes "Decoration" if left empty.